apps/web/src/server/views/validation/review-appeal.njk

```diff
@@ -83,8 +84,8 @@
 	{# TODO: The document links if they are using the same path we might want to build a component that returns the A tag for them #}
 	{{ govukSummaryList({
 		rows: [
-			{ key: { text: "Planning application form" }, value: { html: '<a href="#">planning-application.pdf</a>' } },
-			{ key: { text: "Decision letter" }, value: { html: '<a href="#">decision-letter.pdf</a>' } }
+			{ key: { text: "Planning application form" }, value: { html: '<a href="'+appealData.PlanningApplicationForm.url+'">'+appealData.PlanningApplicationForm.filename+'</a>' } },
```

Comment:
What's the purpose of the '+' in the href and text values here?

Reply:
String concatenation, I am not constructing the HTML in the js file, but here.